src/methods.js

```diff
@@ -0,0 +1,52 @@
+import tasksObj from './tasks.js';
+
+let tasks = tasksObj.getTasks();
+
+function updateTasks() {
+  tasksObj.setTasks(tasks);
+  tasks = tasksObj.getTasks();
+  return tasks;
+}
+
+function addTask(description) {
+  tasks = tasksObj.getTasks();
+  const newTask = {
+    id: tasks.length,
```

Comment:
- [ ] It would be better for the property `index` to be set to the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should equal to 5), currently, your task index start at index zero instead of 1, please update it see screenshot below ⬇️ 

![Screenshot_22](https://user-images.githubusercontent.com/69638013/236394279-86ea891b-d2ec-4fd6-be4b-a7ef6d414c0f.png)

Reply:
OK, thanks for your comment.
It's Done.